scan/src/rpc/OracleScriptHook.re

```diff
@@ -29,6 +29,7 @@ module OracleScript = {
         {
           id: 1,
           oracleScriptID: 1,
+          oracleScriptName: "name",
```

Comment:
Can we use name of oracle script?

Reply:
Just mock for now because it should get from the revision endpoint.